blackbox/docs/sql/information_schema.txt

```diff
@@ -150,6 +150,75 @@ You can even query this tables' own columns (attention: this might lead to infin
   position.
 
 
+The following columns according to the SQL99 standard are available:
+
++--------------------------------+--------------------------------------------+---------------+
+|            Name                |                Description                 |   Data Type   |
++================================+============================================+===============+
+| ``table_catalog``              | Database name containing the table         | ``String``    |
++--------------------------------+--------------------------------------------+---------------+
+| ``table_schema``               | Schema name containing the table           | ``String``    |
++--------------------------------+--------------------------------------------+---------------+
+| ``table_name``                 | Table Name                                 | ``String``    |
++--------------------------------+--------------------------------------------+---------------+
+| ``column_name``                | Column Name                                | ``String``    |
++--------------------------------+--------------------------------------------+---------------+
+| ``ordinal_position``           | Ordinal position of the column starting by | ``Short``     |
+|                                | 1 within the table                         |               |
++--------------------------------+--------------------------------------------+---------------+
+| ``is_nullable``                | ``YES`` if the column is nullable          | ``String``    |
+|                                | ``NO`` if the column is not nullable       |               |
++--------------------------------+--------------------------------------------+---------------+
+| ``data_type``                  | The data type of the column                | ``String``    |
```

Comment:
Could be worthwhile to link to https://crate.io/docs/reference/sql/data_types.html here?